runtime/integration-tests/src/generic/envs/fudge_env/handle.rs

```diff
@@ -96,7 +94,7 @@ pub trait FudgeHandle<T: Runtime> {
 	type RelayApi: BlockBuilderApi<RelayBlock>
 		+ BabeApi<RelayBlock>
 		+ ParachainHost<RelayBlock>
-		+ ApiExt<RelayBlock, StateBackend = <TFullBackend<RelayBlock> as Backend<RelayBlock>>::State>
+		+ ApiExt<RelayBlock>
```

Comment:
I just curious: how did you get to solve the issue that appeared when you changed this?

Reply:
I just removed that and the compiler was happy, the `APIExt` trait only has 1 generic arg now - the block.

Reply:
Wow really? When I removed it, a big big error was shown to me. Maybe something has changes with the rust version

Reply:
Weird, I still get some errors there but they don't seem related to this. Anyway, let's wait until I fix all of these, who knows what dragon will rise ^^